django/core/checks/model_checks.py

```diff
@@ -7,6 +7,7 @@
 
 from django.apps import apps
 from django.core.checks import Error, Tags, register
+from django.db.models import constants
```

Comment:
I think using `from django.db.models.constants import LOOKUP_SEP` will fix the circular import issue causing some test failures.